typescript-function/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "2.4.2",
+  "compilerOptions": {
+      "lib": [
+        "es2017",
+        "esnext.asynciterable"
+      ],
+      "target": "es2017",
+      "module": "commonjs",
```

Comment:
are all those needed?
most of them are defaults anyhow, not?

Reply:
These are literally our own rules from `nodeutils` copy pasted.

Reply:
Consider starting from `tsc --init` since that's the equivalent of `npm init`, `bn create`, etc.

There might be various differences - e.g. targeting node8, not having to publish the module.